targets/targets.json

```diff
@@ -4423,7 +4423,7 @@
             "MX25LM51245G"
         ],
         "components_add": [
-            "OSPIF"
+            "SPINAND"
```

Comment:
Hi
Maybe you have worked with DISCO_L4R9I, but I think this is not supported with default HW ?

Reply:
Yes, default DISCO_L4R9I is not supported SPI NAND Flash.  I replace MX25LM51245G with MX31LF4GE4BC.

Reply:
Hi @jeromecoutant , since Mbed has ended its life, then which RTOS or platforms will new STM32 MCUs be promoted and supported to?

Reply:
Hi
[Zephyr ](https://github.com/zephyrproject-rtos/zephyr) is fully supported for all STM32

https://github.com/zephyrproject-rtos/hal_stm32